arm-web/2015-08-01-preview/swagger/logicManagementClient.json

```diff
@@ -0,0 +1,1299 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "LogicManagementClient",
```

Comment:
Title is required and can be any string since for composite AutoRest clients it is ignored. Calling it LogicManagementClient to differentiate from the other swagger file.